### ISSUE TITLE
extraVolumes & extraVolumesMounts not working

### ISSUE THREAD

**Issue by jamestutton** [elastic/helm-charts#143]
**Chart version:7.1.0**

**Kubernetes version:v1.13.5**

**Kubernetes provider:Rancher RKE**

**Helm Version: Unsure inbuilt in Rancher**

**Values.yaml:**

```
---
extraVolumes: 
  - name: extras
    emptyDir: {}

extraVolumeMounts:
  - name: extras
    mountPath: /usr/share/extras
    readOnly: true
```

**Describe the bug:**

Helm template failed. Error: render error in "elasticsearch/templates/statefulset.yaml": template: elasticsearch/templates/statefulset.yaml:104:14: executing "elasticsearch/templates/statefulset.yaml" at <.Values.extraVolumes>: wrong type for value; expected string; got []interface {} : exit status 1

**Steps to reproduce:**

Attempt to deloy with defaults and the above answers set

**Expected behavior:**

Should merge volume settings into the statefulset.yaml and deploy

**Any additional context:**

**Comment by Crazybus** [comment]
Thanks for reporting this! I'm going to link the draft PR where you already started working on fixing it: https://github.com/elastic/helm-charts/pull/144

**Comment by jmlrt** [comment]
Closing this issue since extraVolumes was fixed in #245.
